config/hubs/schema.yaml

```
@@ -119,6 +119,24 @@ properties:
     then:
       required:
         - stateStore
+  staff:
+    type: object
+    additionalProperties: false
+    description: |
+      List of different login providers staff ids
+    properties:
+      githubIds:
+        type: array
+        description: |
+            List of staff GitHub handles
+        items:
+          type: string
+      googleOauthIds:
+        type: array
```

```suggestion
      github_ids:
        type: array
        description: |
            List of staff GitHub handles
        items:
          type: string
      google_ids:
        type: array
```

to match the new `staff.yaml` keys right?